README.md

```diff
@@ -1,91 +1,129 @@
-- Graph cLISI (`lisi_graph()`)
+- Normalised mutual information `nmi()`
+- Adjusted Rand Index `ari()`
+- Cell type ASW `silhouette_batch()`
+- Isolated label score F1 `isolated_labels()`
+- Isolated label score ASW `isolated_labels()`
+- Cell cycle conservation `cell_cycle()`
+- Highly variable gene conservation `hvg_overlap()`
+- Trajectory conservation `trajectory_conservation()`
+- Graph cLISI `lisi_graph()`
 
 ## Tools
 
 Tools that are compared include:
 
-- [Scanorama](https://github.com/brianhie/scanorama)
-- [scANVI](https://github.com/chenlingantelope/HarmonizationSCANVI)
-- [FastMNN](https://bioconductor.org/packages/batchelor/) (`batchelor_1.4.0`)
-- [scGen](https://github.com/theislab/scgen)
-- [BBKNN](https://github.com/Teichlab/bbknn)
-- [scVI](https://github.com/YosefLab/scVI)
-- [Seurat v3](https://github.com/satijalab/seurat) (3.2.0) CCA and RPCA
-- [Harmony](https://github.com/immunogenomics/harmony) 1.0
-- [Conos](https://github.com/hms-dbmi/conos) 1.3.0
+- [BBKNN](https://github.com/Teichlab/bbknn) 1.3.9
 - [Combat](https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html) [paper](https://academic.oup.com/biostatistics/article/8/1/118/252073)
-- [MNN](https://github.com/chriscainx/mnnpy)
-- [TrVae](https://github.com/theislab/trvae)
-- [DESC](https://github.com/eleozzr/desc)
+- [Conos](https://github.com/hms-dbmi/conos) 1.3.0
+- [DESC](https://github.com/eleozzr/desc) 2.0.3
+- [FastMNN](https://bioconductor.org/packages/batchelor/) (batchelor 1.4.0)
+- [Harmony](https://github.com/immunogenomics/harmony) 1.0
 - [LIGER](https://github.com/MacoskoLab/liger) 0.5.0
+- [MNN](https://github.com/chriscainx/mnnpy) 0.1.9.5
 - [SAUCIE](https://github.com/KrishnaswamyLab/SAUCIE)
-
-## Resources
-
-+ On our [website](https://theislab.github.io/scib-reproducibility) we visualise the results of the study.
-
-+ The reusable pipeline we used in the study can be found in the separate [scIB pipeline](https://github.com/theislab/scib-pipeline.git) repository. It is reproducible and automates the computation of preprocesssing combinations, integration methods and benchmarking metrics.
-
-+ For reproducibility and visualisation we have a dedicated repository: [scib-reproducibility](https://github.com/theislab/scib-reproducibility).
-
-### Please cite:  
-
-_**Benchmarking atlas-level data integration in single-cell genomics.**  
-MD Luecken, M Büttner, K Chaichoompu, A Danese, M Interlandi, MF Mueller, DC Strobl, L Zappia, M Dugas, M Colomé-Tatché, FJ Theis
-bioRxiv 2020.05.22.111161; doi: https://doi.org/10.1101/2020.05.22.111161_ 
+- [Scanorama](https://github.com/brianhie/scanorama) 1.7.0
+- [scANVI](https://github.com/chenlingantelope/HarmonizationSCANVI) (scVI 0.6.7)
+- [scGen](https://github.com/theislab/scgen) 1.1.5
+- [scVI](https://github.com/YosefLab/scVI) 0.6.7
+- [Seurat v3](https://github.com/satijalab/seurat) 3.2.0 CCA (default) and RPCA
+- [TrVae](https://github.com/theislab/trvae) 1.1.2
```

Comment:
I don't think we used 1.1.2 in the paper. We used 0.1.0 or sth, which is no longer avaiable. The 1.1.2 adaptation isn't merged yet.